packages/debugger/test/session.spec.ts

```diff
@@ -228,8 +230,10 @@ describe('protocol', () => {
         frameId
       });
       const scopes = scopesReply.body.scopes;
-      expect(scopes.length).toEqual(1);
-      expect(scopes[0].name).toEqual('Locals');
+      expect(scopes.length).toBeGreaterThanOrEqual(1);
```

Comment:
Something caught on the UI too: https://github.com/jupyterlab/jupyterlab/pull/9346